5.drug-dependency/scripts/1.explore_prism.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Describe PRISM data
+
+# In[1]:
+
+
+import pathlib
+import sys
+
+import pandas as pd
+
+sys.path.append("../")
+from utils import load_utils
+
+
+# In[5]:
```

Comment:
Recommend confirming the the notebook is run sequentially prior to converting to script.